From d7a5917217df3c53bd2746917d9a8f7b691a99ed Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Tue, 24 Dec 2024 04:29:46 +0100
Subject: [PATCH 4/5] Use OpenPGP instead of GnuPG when referring to the
 specification

The name of specification is OpenPGP, and while GnuPG is currently
a widespread implementation, using a specific vendor as if was a
synonym for the specification is a disservice to the whole ecosystem,
more so now that GnuPG upstream has decided to get out of the IETF
working group and not follow the just released OpenPGP RFC.

Use OpenPGP to refer both to the specification and its artifacts.
---
 debian/control   | 10 +++++-----
 debian/copyright |  2 +-
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/debian/control b/debian/control
index 6af283a..0b2f9a4 100644
--- a/debian/control
+++ b/debian/control
@@ -14,9 +14,9 @@ Package: debian-archive-keyring
 Architecture: all
 Multi-Arch: foreign
 Depends: ${misc:Depends}
-Description: GnuPG archive keys of the Debian archive
+Description: OpenPGP archive certificates of the Debian archive
  The Debian project digitally signs its Release files. This package
- contains the archive keys used for that.
+ contains the archive certificates used for that.
 
 Package: debian-archive-keyring-udeb
 Package-Type: udeb
@@ -25,7 +25,7 @@ Architecture: all
 Section: debian-installer
 Depends: ${misc:Depends}
 Recommends: gpgv-udeb
-Description: GnuPG keys of the Debian archive
+Description: OpenPGP archive certificates of the Debian archive
  The Debian project digitally signs its Release files. This package
- contains the archive keys used for that, in a minimal form for use
- in the installer.
+ contains the archive certificates used for that, in a minimal form
+ for use in the installer.
diff --git a/debian/copyright b/debian/copyright
index 4fcb5cb..904a77e 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -1,4 +1,4 @@
-This is Debian GnuPG keyrings of archive keys.
+This is Debian OpenPGP keyrings of archive certificates.
 
 This package was originally put together by Michael Vogt
 <mvo@debian.org> 
-- 
2.47.1

